state/initialize.go

```diff
@@ -160,7 +160,7 @@ func Initialize(args InitializeParams) (_ *Controller, err error) {
 
 	logger.Infof("initializing controller model %s", modelTag.Id())
 
-	modelOps, modelStatusDoc, err := st.modelSetupOps(
+	modelOps, _, err := st.modelSetupOps(
```

Comment:
knit: We now always blackhole the 2nd return result. It can be dropped.

Obviously this is all going anyway, so only bother changing if you need to make other changes